tox.ini

```diff
@@ -0,0 +1,19 @@
+[tox]
+minversion = 3.1
+envlist = py36,py37,pep8
+skipsdist = True
+ignore_basepython_conflict = True
+
+[testenv]
+basepython = python3
+install_command = pip install {opts} {packages}
+deps = -r{toxinidir}/test-requirements.txt
+
+[testenv:linters]
+usedevelop = True
```

Comment:
do we need this option?
locally it complains about missing setup.py
So either we should add setup.py or remove usedevelop = True

Reply:
locally it works for me, and even more it works on zuul

Reply:
it is not required. It is used to control whether test package is installed as link into venv or as a released version. Here there is no project at all